flutter/example/lib/main.dart

```diff
@@ -103,6 +103,13 @@ class MainScaffold extends StatelessWidget {
               onPressed: () async =>
                   {await compute(loop, 10).catchError(handleError)},
             ),
+            // Calling the SDK with an appRunner will handle errors from Futures
+            // in SDKs runZonedGuarded onError handler
+            RaisedButton(
+              child: const Text('Throws in Future.delayed without catchError'),
+              onPressed: () => Future.delayed(Duration(milliseconds: 100),
+                  () => throw Exception('Throws in Future.delayed')),
+            ),
```

Comment:
I still see other buttons calling `catchError` manually, I think they don't need to, if the runZonedGuarded already catches them automatically, so I'd rather remove them (the catchError method call)